hnn_core/gui/gui.py

```diff
@@ -206,10 +205,11 @@ def __init__(self, theme_color="#8A2BE2",
         self.simulation_data = defaultdict(lambda: dict(net=None, dpls=list()))
 
         # Simulation parameters
-        self.widget_tstop = FloatText(value=170, description='tstop (ms):',
-                                      disabled=False)
-        self.widget_dt = FloatText(value=0.025, description='dt (ms):',
-                                   disabled=False)
+        self.widget_tstop = BoundedFloatText(
+            value=170, description='tstop (ms):', min=0, max=1e6,
+            disabled=False)
+        self.widget_dt = BoundedFloatText(
+            value=0.025, description='dt (ms):', min=0, max=10, disabled=False)
```

Comment:
What are the step sizes here?

Reply:
I don't have a good sense for it. Maybe we can set it to 0.025 too?

Reply:
Let's set the `dt` step to 0.01 and  `tstop` step size to 1 as it's more coarse